### PR TITLE
Feature/restaurant list

### PR DESCRIPTION
**Pull request by jasonkang14**
restaurant_list API, which is used to display a list of restaurants on the main page has been completed.
1. display the entire list of restaurants if no category is selected
2. show different restaurants according to the selected category

### CODE REVIEW

File: restaurant/models.py [hunk 1]
```diff
@@ -0,0 +1,14 @@
+from django.db import models
+
+# Create your models here.
+
+class Food_Type(models.Model):
+    name       = models.CharField(max_length=10)
```
Comment (rampart81) [inline restaurant/models.py]: name 과 `=` 사이에 space 하나로 줄여주세요.

File: restaurant/views.py [hunk 2]
```diff
@@ -0,0 +1,17 @@
+import json
+
+from django.shortcuts import render
+from django.http      import JsonResponse, HttpResponse
+from .models          import Restaurant
+from django.views     import View
```
Comment (rampart81) [inline restaurant/views.py]: new line 삽입해주세요 가독성을 위해

File: lunch_buddy/urls.py [hunk 3]
```diff
@@ -18,5 +18,6 @@
 
 urlpatterns = [
 #    path('admin/', admin.site.urls),
-    path('party/', include('group.urls'))
+    path('party/', include('group.urls')),
```
Comment (rampart81) [inline lunch_buddy/urls.py]: 끝에 trailing slash를 안넣어여 url이 `/party` 같은 형태가 가능할것 같아요. Trailing slash를 넣으시면 `/party/` 로만 되서요.

**Comment by jasonkang14** [comment]
change request 완료했습니다!

**Pull request merged**